### PR TITLE
add angelhacks-yyz domain

### PR DESCRIPTION
**Pull request by tregsthedev**
As soon as a vercel project is up for Angelhacks YYZ, this should work

### CODE REVIEW

**Comment by ClaireBookworm** [comment]
I'll add angelhacks-yyz.hackclub.com on the vercel

File: hackclub.com.yaml [hunk 1]
```diff
@@ -2123,10 +2123,14 @@ zh:
   - ttl: 1
     type: CNAME
     value: cname.vercel-dns.com.
-zwp:
+    zwp:
```
Comment (exu3) [inline hackclub.com.yaml:2126]:
```suggestion
zwp:
```

**Comment by tregsthedev** [comment]
@ClaireBookworm can you please change the vercel domain to angelhacks-toronto.hackclub.com

**Comment by tregsthedev** [comment]
That was a misclick, sorry about that Ella!

**Pull request merged**